### PR TITLE
add session-color integration tests

### LINKED ISSUE #2527: Write an integration test for session-color Helper
Parent  issue #1554

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds an integration test for session-color helper

#### Changes proposed in this pull request:

- uses default and specialised case for testing

Fixes #2527

### CODE REVIEW

**Comment by uds5501** [comment]
@CosmicCoder96 Please review!

**Comment by uds5501** [comment]
@CosmicCoder96 Done

File: tests/integration/helpers/session-color-test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    this.set('inputState', 'accepted');
+    await render(hbs`{{session-color inputState}}`);
+    assert.equal(this.element.textContent.trim(), 'yellow');
+
+    this.set('inputState', 'pending');
+    await render(hbs`{{session-color inputState}}`);
+    assert.equal(this.element.textContent.trim(), 'yellow');
+
+    this.set('inputState', '');
```
Comment (CosmicCoder96) [inline tests/integration/helpers/session-color-test.js]: make this 
`this.set('inputState' , 'rejected');`

Comment (uds5501) [reply tests/integration/helpers/session-color-test.js]: Did it

**Comment by uds5501** [comment]
@CosmicCoder96 Done

**Comment by uds5501** [comment]
closed in favour of #2533

**Pull request closed without merging**